Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "UseBaseSpeed", "type": "bool" },
+        { "name": "ChangeRevistivity", "type": "bool" },
+        { "name": "AccelOnlyBoost", "type": "bool" }
+      ]
+    },
+    "ObjOutOfControlCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "EndType", "type": "ObjOutOfControlCollisionSpawner::EndType" },
+        { "name": "EndTime", "type": "float32" }
+      ]
+    },
+    "ObjPhantomCancelCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" },
+        { "name": "CancelType", "type": "ObjPhantomCancelCollisionSpawner::CancelType" }
+      ]
+    },
+    "ObjSetRigidBodySpawner": {
+      "fields": [
+        { "name": "Height", "type": "float32" },
+        { "name": "Width", "type": "float32" },
+        { "name": "Length", "type": "float32" },
+        { "name": "DefaultON", "type": "bool" },
+        { "name": "IsEffect", "type": "bool" },
+        { "name": "IsPlayerTerrain", "type": "bool" },
+        { "name": "m_IsSearchInsulateOnly", "type": "bool" },
+        { "name": "PhysicsType", "type": "ObjSetRigidBodySpawner::PhysicsType" }
+      ]
+    },
+    "ObjSwitchCollisionSpawner": {
+      "fields": [
+        { "name": "TargetList", "type": "array", "subtype": "object_reference" },
+        { "name": "EventTimer", "type": "float32" },
+        { "name": "EventType", "type": "ObjSwitchCollisionSpawner::EventType" },
+        { "name": "TargetList2", "type": "array", "subtype": "object_reference" },
+        { "name": "EventTimer2", "type": "float32" },
+        { "name": "EventType2", "type": "ObjSwitchCollisionSpawner::EventType2" },
+        { "name": "TargetList3", "type": "array", "subtype": "object_reference" },
+        { "name": "EventTimer3", "type": "float32" },
+        { "name": "EventType3", "type": "ObjSwitchCollisionSpawner::EventType3" },
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionDepth", "type": "float32" },
+        { "name": "EventArisingTimer", "type": "float32" },
+        { "name": "SwitchType", "type": "ObjSwitchCollisionSpawner::SwitchType" }
+      ]
+    },
+    "ObjFlowExSpawner": {
+      "fields": [
+        { "name": "EffectID", "type": "uint32" },
+        { "name": "EmitterCount", "type": "uint32" },
+        { "name": "GenerateRatio", "type": "uint32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "AngleX", "type": "float32" },
+        { "name": "AngleY", "type": "float32" },
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and parameter names found on flow objects in Sonic Generations, Sonic Lost World and Sonic Forces. Set data parser function (0x8018BD20, NTSC-J Wii) used for additional help. Parameter names from other games have been updated to match with other parameter names in Sonic Colors.